t/01-make-update-deps.t

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env perl
+# Copyright 2022 SUSE LLC
```

Comment:
Just

```suggestion
# Copyright SUSE LLC
```

for new files

Reply:
on that topic: what license should I write on these files? there's:

```
# SPDX-License-Identifier: MIT
# SPDX-License-Identifier: GPL-2.0-or-later
```

I don't know which one to pic :sweat_smile:

Reply:
It's currently `SPDX-License-Identifier: GPL-2.0-or-later` - not sure why you would want to change it?

Reply:
Just aligning it in all files, no motive in particular.